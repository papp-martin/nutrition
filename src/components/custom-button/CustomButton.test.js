import { shallow } from "enzyme";
import CustomButton from "./custom-button.component";
import React from "react";

it('expect to render CButton component', () => {
    expect(shallow(<CustomButton/>)).toMatchSnapshot();
});