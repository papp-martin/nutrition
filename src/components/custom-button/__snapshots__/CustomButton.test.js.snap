// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CButton component 1`] = `ShallowWrapper {}`;
